jbehave-support-core/src/main/java/org/jbehavesupport/core/report/extension/StepScreenshotsReporterExtension.java

```diff
@@ -0,0 +1,36 @@
+package org.jbehavesupport.core.report.extension;
+
+import java.io.Writer;
+import java.util.Set;
+import org.jbehavesupport.core.TestContext;
+import org.jbehavesupport.core.internal.web.WebScreenshotCreator;
+import org.jbehavesupport.core.report.ReportContext;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class StepScreenshotsReporterExtension extends AbstractXmlReporterExtension {
+
+    private static final String STEP_SCREENSHOTS_REPORTER_EXTENSION = "stepScreenshots";
+    private static final String SCREENSHOT_TAG = "<stepScreenshot>%s</stepScreenshot>\n";
+
+    @Autowired
```

Comment:
Minor detail: constructor injection :slightly_smiling_face:

Reply:
Done, I just had a little problem with it.